pydbtools/utils.py

```diff
@@ -48,22 +65,21 @@ def check_temp_query(sql: str):
         )
 
 
-def clean_query(sql: str, fmt_opts: dict = None) -> str:
+def clean_query(sql: str, fmt_opts: dict = {}) -> str:
     """
     removes trailing whitespace, newlines and final
     semicolon from sql for use with
     sqlparse package
-
     Args:
         sql (str): The raw SQL query
         fmt_opts (dict): Dictionary of params to pass to sqlparse.format.
         If None then sqlparse.format is not called.
     Returns:
         str: The cleaned SQL query
     """
+    fmt_opts["strip_comments"] = True
```

Comment:
I am not sure this should be added by default? Some functions in the codebase are called explicitly without a `fmt_opts` so I assume it is expected to be `None | {}` if not specified.

Reply:
This addresses #49.